Source/WebCore/css/typedom/numeric/CSSMathClamp.cpp

```diff
@@ -76,8 +76,22 @@ void CSSMathClamp::serialize(StringBuilder& builder, OptionSet<SerializationArgu
 
 auto CSSMathClamp::toSumValue() const -> std::optional<SumValue>
 {
-    // FIXME: Implement.
-    return std::nullopt;
+    auto validateSumValue = [](const std::optional<SumValue>& sumValue, const UnitMap* expectedUnits) {
+        return sumValue && sumValue->size() == 1 && (!expectedUnits || *expectedUnits == sumValue->first().units);
```

Comment:
It pains me that this comparison will typically iterate 12 slots to compare two entries. Plus the memory use.

Given that the canonical examples of UnitMap in the spec have only one or two items in the map, I wonder if we should change the HashTraits for UnitMap to make the minimum table size 1 or 2.